Add resource files from ROS World. (#226)
UniversalRobots/Universal_Robots_ROS2_Driver: ur_robot_driver/doc/resources/README.md

# Resources about ROS 2 Universal Robot driver

> The files in this folder are available for use under [![License](https://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/) license.

Reviewer: This image looks a little awkward when inline with text. Can we just saw Creative Common license (and maybe move the image to below the text?)

Reviewer 2: Why Creative Common license?

Author: The idea is to encourage people to use the information from the presentation, but they have to name us on each use.

> The files in this folder are available for use under [![License](https://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/) license.

Author: ```suggestion
The files in this folder are available for use under BSD-3-Clause license.
```

## Diagrams and images

> - [ros2_control diagrams](ros2_control_ur_driver.drawio) - `.drawio` file, Author: [Denis Stogl](mailto:denis.stogl@picknik.ai) (PickNik Robotics)

Reviewer: I tried download the .drawio file and loading it in https://app.diagrams.net/ and got this error

```
Error loading file
Not a diagram file (error on line 15 at column 76: Specification mandates value for attribute crossorigin)
```

Am I doing something wrong? I am not familiar with drawio files.

Author: I just tested it and it works fine.

How did you download the file? I used `wget` and a link to the raw file. Than drag-and-droped it to the diagrams.net

- [generated_images](generated_images/) - see individual files

## 2021-10 ROS World presentation

> [Presentation: Making a robot ROS 2 powered - a case study using the UR manipulators](2021-10_ROS_World_2021_Making_a_robot_ROS2_powered.pdf)

Reviewer: Looks like the pdf compressed down fairly nicely to <5mb. @urrsk would you like this added to the repo?

@destogl this could get updated in the future, maybe we should set up git lfs for .pdfs

Reviewer 2: The size is ok

Summary:

  With the release of ros2_control and MoveIt 2, ROS 2 Foxy finally has all the “ingredients” needed to power a robot with similar features as in ROS 1. We present the driver for Universal Robot’s manipulators as a real-world example of how robots can be run using ROS 2. We show how to realize multi-interface support for position and velocity commands in the driver and how to support scaling controllers while respecting factors set on the teach pendant. Finally, we show how this real-world example influences development of ros2_control to support non-joint related inputs and outputs in its real-time control loop.

Recording: TBA

*Presenter: Denis Štogl*

Authors:

  - Denis Štogl (PickNik Inc.)
  - Dr. Nathan Brooks (PickNik Inc.)
  - Lovro Ivanov (PickNik Inc.)
  - Dr. Andy Zelenak (PickNik Inc.)

>   - Rune Søe-Knudsen (Universal Robots)

Reviewer 2: ```suggestion
  - Rune Søe-Knudsen (Universal Robots A/S)
```

> # Resources about ROS 2 Universal Robot driver

Reviewer: ```suggestion
# Resources about the ROS 2 Universal Robot driver
```

nit
